dll/win32/shell32/CShellLink.cpp

```diff
@@ -2923,6 +2946,15 @@ LRESULT CShellLink::OnNotify(HWND hwndDlg, int idFrom, LPNMHDR pnmhdr)
 
         HeapFree(GetProcessHeap(), 0, unquoted);
 
+
+        m_Header.wHotKey = SendDlgItemMessageW(hwndDlg, IDC_SHORTCUT_KEY_HOTKEY, HKM_GETHOTKEY, 0, 0);
+
+        int idx = (int)SendDlgItemMessageW(hwndDlg, IDC_SHORTCUT_RUN_COMBO, CB_GETCURSEL, 0, 0);
+        if (idx != CB_ERR)
```

Comment:
Is it possible to get `CB_ERR` here?

Reply:
In the unlikely case that something in `OnInitDialog` failed. `LoadString` being the prime candidate but also `CB_ADDSTRING`.

Do you feel it is too defensive?

Reply:
Mostly curiosity.